src/components/common/Dialogs/EditColonyDetailsDialog/EditColonyDetailsDialogForm.tsx

```diff
@@ -123,7 +124,7 @@ const EditColonyDetailsDialogForm = ({
           name="colonyDisplayName"
           appearance={{ colorSchema: 'grey', theme: 'fat' }}
           disabled={disabledInput}
-          maxLength={20}
+          maxLength={MAX_DISPLAY_NAME}
```

Comment:
You can pass this through to the display name input in `StepColonyNameInputs` also, so the user has some visual prompt wrt the character limit

Reply:
Done

![image](https://github.com/JoinColony/colonyCDapp/assets/32598350/2ed33570-ab57-4055-b903-0d94c3efb200)